pages.de/common/eslint.md

```diff
@@ -0,0 +1,20 @@
+# eslint
+
+> Ein erweiterbarer Linter für JavaScript und JSX.
+> Weitere Informationen: <https://eslint.org>.
+
+- Erstellen einer ESLint Konfigurationsdatei:
+
+`eslint --init`
+
+- Linter auf mehreren Dateien ausführen:
+
+`eslint {{dateiname}}.js {{dateiname1}}.js`
+
+- Linterfehler beheben:
+
+`eslint --fix`
+
+- Linter mit einer Konfigurationsdatei ausführen:
+
+`eslint -c {{pfad/zu/konfigurationsdatei}} {{app/src}}`
```

Comment:
```suggestion
`eslint -c {{pfad/zu/konfigurationsdatei}} {{path/to/app_source}}`
```

Mandatory fix (requires the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/eslint.md) page to be changed): [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths) path format is used.

Reply:
@pixelcmtd, without `_` I get this translation: `Pfad/zu/App-Quelle`. Should I change my suggestion?

Reply:
I think I'd use `pfad/zu/quellordner` and `path/to/source_directory`, how about that?

Reply:
I've changed my suggestion to reflect your suggestion. :)